features/webvtt-cue-settings.yml

```diff
@@ -13,6 +13,7 @@ compat_features:
   - api.VTTCue.align
   - api.VTTCue.line
   - api.VTTCue.position
+  - api.VTTCue.region
```

Comment:
I think this belongs with `webtt-regions`. The type of `region` is `VTTRegion`, so it seems kinda weird that it's not with the rest of that API.